R/google_trends.R

```diff
@@ -0,0 +1,89 @@
+#' Access google trends
+#' @param user - User id.
+#' @param password - Password for the account.
+#' @param query - Vector of queries.
+#' @param type - Output type. Can be "top_regions", "top_cities" or "trends".
+#' @param last - From when the data should be retreived.
```

Comment:
Could you add list of possible values ( "1h", "5y" etc.) so that user of this function does not need to go through the code for the list?